docs/source/basics.rst

```diff
@@ -170,19 +170,94 @@ That's all we have to do to declare the memory. Continue to the next section on
 using ``ro`` to store measured readout results. Check out :ref:`parametric_compilation` to see how you might use
 ``theta`` to compile gate parameters dynamically.
 
-
 .. _measurement:
 
 Measurement
 ~~~~~~~~~~~
 
-*Coming soon*
+There are several ways you can handle measurements in your program. We will start with the simplest method -- letting
+the ``QuantumComputer`` abstraction do it for us.
+
+.. code:: python
+
+    from pyquil import Program, get_qc
+
+    # Get our QuantumComputer instance, with a Quantum Virutal Machine (QVM) backend
+    qc = get_qc("8q-qvm")
+
+    # Construct a simple Bell State
+    p = Program("H 0", "CNOT 0 1")
```

Comment:
I think we should match the style with the programs below. We should import `H` and `CNOT`.